services/director-v2/tests/unit/with_dbs/conftest.py

```diff
@@ -5,52 +5,346 @@
 # pylint:disable=no-name-in-module
 
 
-from typing import Any, Callable, Dict, List
+import json
+from datetime import datetime
+from typing import Any, Callable, Dict, Iterator, List
+from uuid import uuid4
 
 import pytest
+import sqlalchemy as sa
 from _helpers import PublishedProject, RunningProject  # type: ignore
+from faker import Faker
+from models_library.clusters import Cluster
 from models_library.projects import ProjectAtDB
-from simcore_postgres_database.models.comp_pipeline import StateType
+from models_library.projects_nodes_io import NodeID
+from pydantic.main import BaseModel
+from simcore_postgres_database.models.cluster_to_groups import cluster_to_groups
+from simcore_postgres_database.models.clusters import clusters
+from simcore_postgres_database.models.comp_pipeline import StateType, comp_pipeline
+from simcore_postgres_database.models.comp_runs import comp_runs
+from simcore_postgres_database.models.comp_tasks import comp_tasks
+from simcore_postgres_database.models.projects import ProjectType, projects
+from simcore_postgres_database.models.users import UserRole, UserStatus, users
 from simcore_service_director_v2.models.domains.comp_pipelines import CompPipelineAtDB
 from simcore_service_director_v2.models.domains.comp_runs import CompRunsAtDB
-from simcore_service_director_v2.models.domains.comp_tasks import CompTaskAtDB
+from simcore_service_director_v2.models.domains.comp_tasks import CompTaskAtDB, Image
+from simcore_service_director_v2.utils.computations import to_node_class
+from simcore_service_director_v2.utils.dask import generate_dask_job_id
+from sqlalchemy.dialects.postgresql import insert as pg_insert
+
+
+@pytest.fixture()
+def registered_user(
```

Comment:
or just to an upper level contest.py file

Reply:
so the point is I purposedly moved it out of the upper level conftest.py file because:
- this is a database dependency, which should NOT be used by any test in the not db part.

Reply:
I will move it to pytest-simcore